lesson17/index.html

```diff
@@ -5,17 +5,14 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>lesson 16</title>
+  <title>lesson 17</title>
   <link rel="stylesheet" href="style.css">
+  <script src="./index.js" defer></script>
 </head>
 
 <body id="js-body">
```

Comment:
Q
I don't think this id is being used, is it necessary?

Reply:
[a55fe0e](https://github.com/hippopo-moo/js-handson-work/pull/19/commits/a55fe0e7340deacf2946ddada4d45165a7d8b491)

It was a completely unnecessary id😫! Deleted.